Wireframe/index.html

```diff
@@ -1,13 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <link rel="stylesheet" href="/Wireframe/css/style.css">
+</head>
+
+<body>
+    <header>
+        <section class="title">
+            <h1>Intoduction to Git</h1>
+    
+        </section>
+
+        <section class="hero">
+
+            <div class="hero__card">
+                <h2 class="hero__title">Git is used for</h2>
+                <pclass="hero__description">
+                    Git is a popular version control system. It was created by Linus Torvalds in 2005, and has been maintained by Junio Hamano since then.
+                    Tracking code changes.
+                    Tracking who made changes.
+                    Coding collaboration.
+                    </p>
+                    <button class="hero__button">
+                        Read More
+                    </button>
+            </div>
+            <img src="/Wireframe/git-hero.jpg" alt="hero-bg">
```

Comment:
This alt text will be read to a screenreader. What makes a good alt text? Google it and let me know what you find.

Reply:
The helpful article provides tips on writing effective alt text.
https://accessibility.huit.harvard.edu/describe-content-images#:~:text=Alternative%20(Alt)%20Text%20is%20meant,example%20of%20a%20missing%20image.

Reply:
So what would you change this to?